starters/deno-oak-denodb/README.md

```diff
@@ -192,6 +192,35 @@ This starter kit has the following structure in the src folder:
 - `rest` contains only the health endpoint. Should you need to use a REST API instead, you can use this health endpoint as an example for building a REST API.
 - `util` contains various utility functions.
 
+### Default API routes
+
+The starter kit comes with a REST API for the healthcheck and a GraphQL API for everything else.
+
+The REST API's healtcheck is available at http://localhost:3333/health by default and returns the status of the databases.
+
+The GraphQL API is available at http://localhost:3333/graphql by default. If you execute a GET on it, you will get the GraphQL playground (in non-production only). If you execute a POST, you can use the GraphQL API. The API has the following operations:
+
+- `getTechnologies` gets you all technologies in the database
+- `getTechnology` gets you a single technology from the database
```

Comment:
Changed to `technologies` and `technology`

Reply:
Ah, I was working on an old version of the branch. Thanks.